experiment/references.md

```diff
@@ -1 +1,12 @@
-### Link your references in here
\ No newline at end of file
+
+
+   
+[//]: # (Non functioning link : https://www.nanoscape.northwestern.edu/Vol2/04Hall.pdf)
+   
+[//]: # (Non functioning link : https://www.nanohub.org/site/wiki/436/Silvergold_module.pdf)
```

Comment:
The links reported on the broken link Google sheet look different from these. Please keep them in sync.